src/jdk.jdi/share/classes/com/sun/jdi/ThreadReference.java

```diff
@@ -125,9 +125,10 @@ public interface ThreadReference extends ObjectReference {
      * @param throwable the asynchronous exception to throw
      * @throws InvalidTypeException if <code>throwable</code> is not
      * an instance of java.lang.Throwable in the target VM
-     * @throws IllegalThreadStateException if the thread has terminated,
-     * or the thread is a virtual thread that cannot be stopped with an
-     * asynchronous exception at this time
+     * @throws IllegalThreadStateException if the thread has terminated
+     * @throws java.lang.UnsupportedOperationException if the thread is a
+     * virtual thread that cannot be stopped with an asynchronous exception
```

Comment:
Minor comment is that you can drop java.lang if you want as that packaged is always imported.